src/condor_schedd.V6/schedd.cpp

```diff
@@ -15687,10 +15702,11 @@ Scheduler::unlinkMrec(match_rec* match)
 	jobId.cluster = match->cluster;
 	jobId.proc = match->proc;
 
-	// If this match is an ocu holder, make sure to update the
+	// If this match was an ocu holder, make sure to update the
 	// correct job.
 	if (match->is_ocu && jobId.cluster == -1 && jobId.proc == -1) {
 		jobId = match->ocu_originator;
+		SetAttributeInt(jobId.cluster, jobId.proc, ATTR_OCU_CLAIMED_TIME, time(nullptr) - match->entered_current_status);
```

Comment:
This will stomp any existing value of ATTR_OCU_CLAIMED_TIME, rather than incrementing it. That seems wrong.